src/families/stellar/libcore-signOperation.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+import Stellar from "@ledgerhq/hw-app-str";
+import type { CoreStellarLikeTransaction, Transaction } from "./types";
+import { makeSignOperation } from "../../libcore/signOperation";
+import {
+  libcoreAmountToBigNumber,
+  libcoreBigIntToBigNumber
+} from "../../libcore/buildBigNumber";
+import buildTransaction from "./libcore-buildTransaction";
+
+async function signTransaction({
+  account: { freshAddress, freshAddressPath, balance, id },
+  transport,
+  transaction,
+  coreTransaction,
+  isCancelled,
+  onDeviceSignatureGranted,
+  onDeviceSignatureRequested
+}) {
+  // Sign with the device
+  const hwApp = new Stellar(transport);
+  const serialized = await coreTransaction.toSignatureBase();
+  onDeviceSignatureRequested();
+  const { signature } = await hwApp.signTransaction(
+    freshAddressPath,
+    Buffer.from(serialized, "hex")
+  );
+  onDeviceSignatureGranted();
+
+  if (isCancelled()) return;
+
+  await coreTransaction.putSignature(
+    signature.toString("hex"),
+    await coreTransaction.getSourceAccount()
+  );
+  if (isCancelled()) return;
+
+  const hex = await coreTransaction.toRawTransaction();
+  if (isCancelled()) return;
+
+  const recipients = [transaction.recipient];
+  const senders = [freshAddress];
+
+  const feesRaw = await coreTransaction.getFee();
+  if (isCancelled()) return;
+
+  const fee = await libcoreAmountToBigNumber(feesRaw);
+  if (isCancelled()) return;
+
+  const transactionSequenceNumberRaw = await coreTransaction.getSourceAccountSequence();
+  const transactionSequenceNumber = (
+    await libcoreBigIntToBigNumber(transactionSequenceNumberRaw)
+  ).toNumber();
+
+  // Make an optimistic response
+
+  let op;
+
+  const txHash = ""; // resolved in broadcast()
```

Comment:
🤔 😆

Reply:
I took it from an another coin, but yeah...